src/RA_Defs.h

```diff
@@ -159,15 +165,22 @@ class ResizeContent
     };
 
 public:
-    HWND hwnd;
-    POINT pLT;
-    POINT pRB;
-    AlignType nAlignType;
-    int nDistanceX;
-    int nDistanceY;
-    bool bResize;
-
-    ResizeContent(HWND parentHwnd, HWND contentHwnd, AlignType newAlignType, bool isResize)
+    HWND hwnd{ nullptr };
+    POINT pLT = POINT{};
+    POINT pRB = POINT{};
```

Comment:
For stuff like this, just hover over the '=' sign to see why I did it. No implicit knowledge necessary, just knowledge of C++.